apps/mull-ui/src/app/pages/event-page/event-page.tsx

```diff
@@ -26,8 +26,8 @@ export const EventPage = ({
   const { id } = useParams<{ id: string }>();
   const eventId = parseInt(id);
 
-  const { loading, error, data } = useFindSpecificEventQuery({
-    variables: { eventId },
+  const { loading, error, data } = useEventQuery({
```

Comment:
```suggestion
  const { loading, error, data } = useFindEventQuery({
```

Reply:
No worries, if we change the names I'll take care of making sure everything is renamed properly.